symbulate/tests/test_distributions.py

```diff
@@ -10,25 +10,705 @@
 class TestBernoulli(unittest.TestCase):
 
     def test_p_one(self):
-        X = RV(Bernoulli(1))
+        X = RV(Bernoulli(p=1))
         sims = X.sim(Nsim)
         self.assertTrue(all(sim == 1 for sim in sims))
-    
+   
     def test_sum(self):
-        X = RV(Bernoulli(.4) ** 5)
+        exp_list, obs_list = [], []
+        X = RV(Bernoulli(p=.4) ** 5)
         sims = X.apply(sum).sim(Nsim)
-        obs = list(sims.tabulate().values())
-        exp = Nsim * stats.binom(n=5, p=.4).pmf(range(6))
-        pval = stats.chisquare(obs, exp).pvalue
+        key_list = sims.tabulate()
```

Comment:
This needs to be renamed. `key_list` is not a descriptive name for this variable.

Please fix everywhere below.